src/sprite/sprite_data.cpp

```diff
@@ -127,6 +128,7 @@ SpriteData::parse_action(const ReaderMapping& mapping)
         throw std::runtime_error("hitbox should specify 2/4 coordinates");
     }
   }
+  mapping.get("unisolid", action->hitbox_unisolid);
```

Comment:
Does this need a default?

Reply:
It's already `false`, check the `SpriteData::Action` constructor.